src/deprecations.json

```diff
@@ -1,16 +1,4 @@
 {
-  "cssCustomProperties": [
-    "--nds-black",
-    "--nds-grey",
-    "--nds-medium-grey",
-    "--nds-lightest-grey",
-    "--nds-smoke-grey",
-    "--nds-red",
-    "--nds-white",
-    "--nds-font-family",
-    "nds-header-font",
-    "subdued-20-opacity",
-    "subdued-10-opacity"
-  ],
-  "components": ["Dropdown"]
+  "cssCustomProperties": [],
+  "components": []
```

Comment:
This file exists so that consumers can create their own danger rules and/or lint warnings when deprecated components or vars are used.